### ISSUE TITLE
Problem with Namespaced controllers in 1.2

### ISSUE THREAD

**Issue by diegorv** [activeadmin/inherited_resources#117]
Hi,

In 1.1.2, I have controllers like that:
    class Admin::UsersController < InheritedResources::Base
    end

After update for 1.2, it doesnt work anymore.

I need change for

```
class Admin::UsersController < InheritedResources::Base
    defaults :resource_class => User
end
```

It`s a bug? My problem?

Because in readme, i read that:

Namespaced controllers works out the box.
Inherited Resources prioritizes the default resource class for the namespaced controller in this order:

Forum::Post
ForumPost
Post

**Comment by josevalim** [comment]
Seems to be a bug, maybe you can work on a patch? :)

**Comment by sirupsen** [comment]
This seems odd, except if you have a model called either Admin::User or AdminUser, which I assume you don't. I just created [a test for your exact case](https://github.com/Sirupsen/inherited_resources/commit/f856f1833b35f5158159dcdd9154412ff9357bb6), and this test passes.

**Comment by diegorv** [comment]
Sirupsen,

My model is just "User".

In my view i get "User" for  <%= debug(@controller.resource_class) %>

Without "defaults :resource_class => User", I can enter in http://localhost:3000/admin/users, but after fill the fields and submit the form, i get "empty fields" for all the validations.

With "defaults :resource_class => User", after submit, no problem with "empty fields"

:/

**Comment by josevalim** [comment]
A-ha, so the issue seems to be related with the request_name used.

Btw, @Sirupsen thanks for adding the feature and for investigating it now. :)

**Comment by sirupsen** [comment]
Aha, I will look into this tomorrow!

Jose, no problem!

**Comment by sirupsen** [comment]
[Fixed](https://github.com/josevalim/inherited_resources/pull/120)

**Comment by sirupsen** [comment]
Request param troubles in namespaced controllers fixed, closed by 283b29bc6248e95f9f46148ca5eb49b2bc52c893
